### DISCUSSION TITLE
Can you share the pick and place file?

### DISCUSSION THREAD

**Discussion by laleeroy** [tjko/fanpico#84] [Q&A]
I want to order pre-assembled pcb, I found the gerber and bom already but can't seem to find the PickAndPlace file

**Comment by tjko** [comment]
I haven't included it in the repository, since I've been having issues with the files generated from KiCad.  Not all components have been oriented correctly, PCB manufacturer had to fix some components (possibly this might be PCB manufacture related issue, but not sure).

Attached is the footprint position file from KiCad for the current 0804D v1.2c PCB:
[top-pos-v1.2c.csv](https://github.com/tjko/fanpico/files/15302644/top-pos-v1.2c.csv)
(be warned that some components may not be correctly oriented and may need to fixed)

**Comment by tjko** [comment]
Attached is the position file from KiCad for the current 0804D v1.3 PCB:
[top-pos-v1.3.csv](https://github.com/user-attachments/files/20116177/top-pos-v1.3.csv)

There still seems to be issue with some component not being "rotated" correctly requiring fixing it before assembly (at least this was happening with JLCPCB).  Not sure if issue is with KiCad side or on the PCB manufacturer side?

**Reply by Metritutus** [reply]
> Attached is the position file from KiCad for the current 0804D v1.3 PCB: [top-pos-v1.3.csv](https://github.com/user-attachments/files/20116177/top-pos-v1.3.csv)
> 
> There still seems to be issue with some component not being "rotated" correctly requiring fixing it before assembly (at least this was happening with JLCPCB). Not sure if issue is with KiCad side or on the PCB manufacturer side?

If using JLCPCB, have you tried using the [Fabrication Toolkit](https://github.com/bennymeg/Fabrication-Toolkit) plugin for the export?  Googling the rotation issue led me to it.  I had a quick look and the rotations of its output do appear different to those exported via KiCad's Component Placement modal, however I think it also includes the non "SMD footprints"(?), and I don't know if the new rotations are correct either (I'm not sure how to go about verifying them as this is all quite new to me).

**Reply by tjko** [reply]
Thanks, had not noticed that before. I'll take a look when next time submit JLCPCB order.

With JLCPCB, when you submit an assembly order it provides you screens to verify the component placement. Where you can fix incorrectly placed components (by fixing the rotation).

**Comment by tjko** [comment]
Attached is the footprint position file from KiCad for the current 0401D v1.1 PCB:
[0401D_top-posv1.1.csv](https://github.com/user-attachments/files/21333509/0401D_top-posv1.1.csv)
(be warned that some components may not be correctly oriented and may need to fixed)
